### ISSUE TITLE
Regarding the model used in the paper "Personalizing Dialogue Agents"

### ISSUE THREAD

**Issue by dimeldo** [facebookresearch/ParlAI#540]
Hi, could you elaborate on the parameters used for the generative model in the [paper](https://arxiv.org/abs/1801.07243)? @Saizheng @emilydinan @JackUrb
I would like to know about:

1. Cell type (LSTM probably as stated in the paper?)
2. No. of units (encoder, decoder).
3. No. of layers (encoder, decoder).
4. Learning rate.
5. Embeddings size.
6. Optimizer (Adam, SGD).
7. Batch size.
8. No. of epochs until convergence.

Thanks a lot!

**Comment by Saizheng** [comment]
hello,
this is the model we used:
1. lstm
2. 1024 for both
3. 1 layer for both
4.1e-3
5. 300, using https://github.com/stanfordnlp/GloVe
6. Adam
7. 128
8. we use early stopping and monitoring the hit@1 in validation set every 300s, we stopped training if the validation hit@1 does not increase after 10 * 300s.

**Comment by jaseweston** [comment]
we plan to get this as a reproducible setup in parlAI soon..

**Comment by dimeldo** [comment]
Thank you very much!
